ui/components/ui/tabs/dropdown-tab/dropdown-tab.component.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
```

Comment:
Please remove the .component extension in the filename, this is an old pattern from a time  when we had .containers too but now that we use hooks we do not create container files.